crates/compiler/input/add.cairo

```diff
@@ -0,0 +1,10 @@
+use core::felt252;
+
+fn _main() -> felt252 {
+            let n = 2 + 3;
+            n
+}
+
+fn main() -> felt252 {
+  _main()
+}
```

Comment:
Please format this consistently using the same conventions as our Rust. Right now it has really wonky indentation.